dask_geopandas/core.py

```diff
@@ -0,0 +1,330 @@
+            yoff=yoff,
+            zoff=zoff,
+            enforce_metadata=False,
+        )
+
+    @derived_from(geopandas.base.GeoPandasBase)
+    def rotate(self, angle, origin="center", use_radians=False):
+        return self.map_partitions(
+            self._partition_type.rotate,
+            angle,
+            origin=origin,
+            use_radians=use_radians,
+            enforce_metadata=False,
+        )
+
+    @derived_from(geopandas.base.GeoPandasBase)
+    def scale(self, xfact=1.0, yfact=1.0, zfact=1.0, origin="center"):
+        return self.map_partitions(
+            self._partition_type.scale,
+            xfact=xfact,
+            yfact=yfact,
+            zfact=zfact,
+            origin=origin,
+            enforce_metadata=False,
+        )
+
+    @derived_from(geopandas.base.GeoPandasBase)
+    def skew(self, xs=0.0, ys=0.0, origin="center", use_radians=False):
+        return self.map_partitions(
+            self._partition_type.skew,
+            xs=xs,
+            ys=ys,
+            origin=origin,
+            use_radians=use_radians,
+            enforce_metadata=False,
+        )
+
+    @derived_from(geopandas.base.GeoPandasBase)
+    def explode(self):
+        return self.map_partitions(self._partition_type.explode, enforce_metadata=False)
+
+    @property
+    def cx(self):
+        raise NotImplementedError
+
+
+class GeoSeries(_Frame, dd.core.Series):
+    _partition_type = geopandas.GeoSeries
+
+
+class GeoDataFrame(_Frame, dd.core.DataFrame):
+    _partition_type = geopandas.GeoDataFrame
+
+
+from_geopandas = dd.from_pandas
+
+
+def from_dask_dataframe(df):
+    return df.map_partitions(geopandas.GeoDataFrame)
+
+
+def points_from_xy(df, x="x", y="y", z="z"):
+    """Convert dask.dataframe of x and y (and optionally z) values to a GeoSeries."""
+
+    def func(data, x, y, z):
+        return geopandas.GeoSeries(
+            geopandas.points_from_xy(
+                data[x], data[y], data[z] if z in df.columns else None
+            )
+        )
+
+    return df.map_partitions(func, x, y, z, meta=geopandas.GeoSeries())
+
+
+for name in [
+    "area",
+    "geom_type",
+    "type",
+    "length",
+    "is_valid",
+    "is_empty",
+    "is_simple",
+    "is_ring",
+    "has_z",
+    "boundary",
+    "centroid",
+    "convex_hull",
+    "envelope",
+    "exterior",
+    "interiors",
+    "bounds",
+]:
+    _Frame._bind_property(name)
+
+for name in [
+    "geometry",
+    "x",
+    "y",
+]:
+    GeoSeries._bind_property(name)
+
+for name in [
+    "contains",
+    "geom_equals",
+    "geom_almost_equals",
+    "crosses",
+    "disjoint",
+    "intersects",
+    "overlaps",
+    "touches",
+    "within",
+]:
+    meth = getattr(geopandas.base.GeoPandasBase, name)
+    GeoSeries._bind_elemwise_comparison_method(
+        name, meth, original=geopandas.base.GeoPandasBase
+    )
+    GeoDataFrame._bind_elemwise_comparison_method(
+        name, meth, original=geopandas.base.GeoPandasBase
```

Comment:
Why do those need to be bound to both GeoSeries and GeoDataFrame (instead of once to `_Frame`)

Reply:
Binding to `_Frame` works fine. I think this is just a copy-paste hold-out from dask-cudf.